### PR TITLE
Add a argparse to accept command line parameters

### PR DESCRIPTION
**Pull request by csachs**
Thanks for merging #11. Here another split off from #10 .  (Plus additionally allowing `latest` as an version argument now).
Allows to pass arguments via command line. Certainly no necessity, but might be nice.

### CODE REVIEW

File: make_wheels.py [hunk 1]
```diff
@@ -129,50 +131,65 @@ def fetch_zig_version_info():
...
+        with open(wheel_path, 'rb') as wheel:
+            print(f'  {hashlib.sha256(wheel.read()).hexdigest()} {wheel_path}')
+
+def get_argparser():
+    parser = argparse.ArgumentParser(prog=__file__, description="Repackage official Zig downloads as Python wheels")
+    parser.add_argument('--version', default='latest',
+                        help="version to package, use `latest` for latest release, `master` for nightly build")
+    parser.add_argument('--suffix', default='', help="wheel version suffix")
+    parser.add_argument('--outdir', default='dist/', help="target directory")
+    parser.add_argument('--platform', action='append', choices=list(ZIG_PYTHON_PLATFORMS.keys()), default=[],
```
Comment (whitequark) [inline make_wheels.py:185]: I'd like the default to stay the same as before.

Comment (whitequark) [reply make_wheels.py:191]: Sorry, I actually mean building all of the platforms if none are specified.

Comment (csachs) [reply make_wheels.py:191]: Added.
I guess with a bit of Github Action automation, packaging and pushing new versions to PyPI could be automated.

Comment (csachs) [reply make_wheels.py:191]: Ah sorry, it should do that already (or before the second commit). Line 139, if no platforms are specified, it iters thru all.

Comment (whitequark) [reply make_wheels.py:191]: Ah okay, I misread then. I think defaulting to latest is fine then.

For GitHub automation, take a look at the YoWASP project which does exactly that.

Comment (csachs) [reply make_wheels.py:185]: Okay, thanks. I'll take a look.

I've removed the last commit again.

**Comment by whitequark** [comment]
Sorry for the delay, and thank you for the PR!

**Pull request merged**